string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,23 @@
+/* exported numVowels */
+/* create storage space for output starting at 0
+go through each letter in the string and check if it is a vowel
+if yes, increase output by 1
+return output */
+function numVowels(string) {
+  var count = 0;
+  for (var i = 0; i < string.length; i++) {
+    if (string[i] === 'a' ||
+    string[i] === 'e' ||
+    string[i] === 'i' ||
+    string[i] === 'o' ||
+    string[i] === 'u' ||
+    string[i] === 'A' ||
+    string[i] === 'E' ||
+    string[i] === 'I' ||
+    string[i] === 'O' ||
+    string[i] === 'U') {
```

Comment:
If you convert the string to either lower case or upper case before the loop, you can cut down the number of expressions in your condition by half 🤓